test/parallel/test-crypto-dhparam-warning.js

```diff
@@ -0,0 +1,23 @@
+// Flags: --no-warnings
+'use strict';
+
+const common = require('../common');
+
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
+
+const tls = require('tls');
+
+common.expectWarning('SecurityWarning',
+                     'DH parameter is less than 2048 bits');
+
+const raw =
+  '-----BEGIN DH PARAMETERS-----\n' +
+  'MIGHAoGBAUwAFaR7qYWwAsvbBSsronNFncoOjp0CwTCSgpyUxOBUSBd6PQF2D8YB\n' +
+  'gT3AkMuLWh2vhKnH6LM8F5dXmOX6sa1WZ3WT+xCRpU+biOJPq7EvuTq1KsXXeKwn\n' +
+  'io5yh83Oa/ILvopaf7KrN8tWbJRq0iAIsSYWgy/sK/pRx5dVeGO7AgEC\n' +
+  '-----END DH PARAMETERS-----';
+
```

Comment:
This is 1025 bits dhparam as below.
````
Diffie-Hellman-Parameters: (1025 bit)
    prime:
        01:4c:00:15:a4:7b:a9:85:b0:02:cb:db:05:2b:2b:
        a2:73:45:9d:ca:0e:8e:9d:02:c1:30:92:82:9c:94:
        c4:e0:54:48:17:7a:3d:01:76:0f:c6:01:81:3d:c0:
        90:cb:8b:5a:1d:af:84:a9:c7:e8:b3:3c:17:97:57:
        98:e5:fa:b1:ad:56:67:75:93:fb:10:91:a5:4f:9b:
        88:e2:4f:ab:b1:2f:b9:3a:b5:2a:c5:d7:78:ac:27:
        8a:8e:72:87:cd:ce:6b:f2:0b:be:8a:5a:7f:b2:ab:
        37:cb:56:6c:94:6a:d2:20:08:b1:26:16:83:2f:ec:
        2b:fa:51:c7:97:55:78:63:bb
    generator: 2 (0x2)
-----BEGIN DH PARAMETERS-----
MIGHAoGBAUwAFaR7qYWwAsvbBSsronNFncoOjp0CwTCSgpyUxOBUSBd6PQF2D8YB
gT3AkMuLWh2vhKnH6LM8F5dXmOX6sa1WZ3WT+xCRpU+biOJPq7EvuTq1KsXXeKwn
io5yh83Oa/ILvopaf7KrN8tWbJRq0iAIsSYWgy/sK/pRx5dVeGO7AgEC
-----END DH PARAMETERS-----
````
We already have 1024 bits key in `test/fixtures/keys/dh1024.pem`.   Is there any reason to use 1025 bits dh param for this test?  Otherwise, I think this test can be merged by adding `common.expectWarning` check in `test/parallel/test-tls-dhe.js`